### ISSUE TITLE
Overfitting?

### ISSUE THREAD

**Issue by djanibekov** [iamsardorbek/Body-Performace-Data-Analysis#3]
Look at metrics for trained data and compare with test data. If there is a big gap your model is likely overfitted

**Comment by iamsardorbek** [comment]
![image](https://user-images.githubusercontent.com/58874777/153667121-b8c670a3-c473-4bbe-a1de-a1cb32e1f575.png)
To my perspective, the model does not suffer from either overfitting or underfitting thanks to taking away 10% of the dataset for testing and increasing the random_state to 7.

**Comment by djanibekov** [comment]
ok, `random_state`  change the distribution of training and testing datasets, but increasing or decreasing it doesn't affect the performance of a model. It is used to save the state how the initial dataset was split into train-test to have an opportunity to iterate model fitting process again.

Accuracy computed - is it testing dataset? (If yes, your model did overfit on the training data. Ideally, your train and test accuracies should be equal)
